### DISCUSSION TITLE
Gopro 12 and powerbank

### DISCUSSION THREAD

**Discussion by OmikeOo** [gopro/labs#918] [General]
Really struggling here. Have a new gp12 and an Anker 20000w powerbank with an Gen 2 USB cable. I've installed the GP labs firmware and used the TUSB=1 QR code but still if I try to turn on the GP without a battery installed I get the low power message. Am I missing something. This really seems to me to be a major issue with the GP which had I known about earlier would have stopped me from buying it.

**Comment by dnewman-gpsw** [comment]
Use *TUSB=1,  without the preceding * or the older !M, it will not work. Too many users abbreviate the commands in discussions. The command documentation is here https://gopro.github.io/labs/control/tech/

PS something in the power supply or cable is not reporting the current rating correctly.  It is annoying.  *TUSB=1 will stop requirement the current rating being reported.  It still requires the supply to be 2.4A or better, which I expect yours is.

**Comment by OmikeOo** [comment]
Thanks again David, That appears to be working. Now awaiting a new, better
quality 512gb SD card to try, and then see if I need a new laptop to be
able to play back the files! My Old, (original) GoPro really was very
simple!

Mike
